azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/SetupControlBarView.kt

```diff
@@ -129,6 +129,10 @@ internal class SetupControlBarView : LinearLayout {
         audioDeviceButton.refreshDrawableState()
     }
 
+// <<<<<<< HEAD
```

Comment:
We can remove this comment Adam

Reply:
I'll follow up in the second PR, (with wired headphone) don't want to clear approvals and chase for a day